packages/api/templates/index.html

```diff
@@ -25,10 +25,20 @@
       .then((isConnected) => {
         if (isConnected) {
           const payment = {
-            amount: '20',
+            amount: '0.2',
             destination: issuer || 'rNvFCZXpDtGeQ3bVas95wGLN6N2stGmA9o',
             currency,
-            issuer
+            issuer,
+            memos: [
+              {
+                Memo: {
```

Comment:
@FlorianBouron This is the Memo type directly from xrpljs, hence the capital letters for each property. Should we write our own type without the capital and then do a translation?

Reply:
Changed in [this commit](https://github.com/GemWallet/gemwallet-extension/pull/152/commits/9e38863eecc40c8fc683e636a4ffc206994d6c5c)